tekton/pipeline_run.go

```diff
@@ -19,13 +19,15 @@ package tekton
 import (
 	"encoding/json"
 	"fmt"
+	"os"
+	"unicode"
+
 	libhandler "github.com/operator-framework/operator-lib/handler"
+	appstudioshared "github.com/redhat-appstudio/managed-gitops/appstudio-shared/apis/appstudio.redhat.com/v1alpha1"
```

Comment:
Could we use the same name used in the main file? That's it, `gitopsv1alpha1`.

Reply:
I'm ok with you using `appstudioshared` as the GitOps service have API definitions in at least 3 different places, but please add the API version to the name (`appstudiosharedv1alpha1) so it's clear what version are we using.

This PR is blocking progress in e2e, so let's get it changed and merged as soon as possible.

Reply:
It is written exactly how the gitops team says to write it https://github.com/redhat-appstudio/managed-gitops/tree/main/appstudio-shared#remember-to-add-the-scheme-to-your-operators-maingo 
There is no reason to not follow their documentation

Reply:
My concerns are that the section in the README is only an example and not using a versioned import will only make more difficult to follow code specially during reviews. It will also require changes in the case of using multiple versions. In addition it breaks with the strategy followed up to this point in which every single CRD import is using a versioned name.

Having said that, let's move on to unblock progress on e2e and reevaluate in the future.